homeassistant/components/zwave_js/discovery.py

```diff
@@ -1170,6 +1170,16 @@ def async_discover_single_value(
         yield from async_discover_single_configuration_value(
             cast(ConfigurationValue, value)
         )
+    elif value.metadata.stateful is False:
+        yield ZwaveDiscoveryInfo(
```

Comment:
Is there never a case of us only supporting an event entity and no other platform for a device?

Reply:
I'll switch to a normal discovery schema just in case